flake8_isort.py

```diff
@@ -10,7 +10,7 @@
 import warnings
 
 
-__version__ = '4.1.2.post1.dev0'
+__version__ = '4.2.0'
```

Comment:
Please leave this as it is, when a new release is cut it is automatically handled by `zest.releaser`

Reply:
@gforcada I revert it